dependencies/windows/install-dependencies.cmd

```diff
@@ -2,14 +2,14 @@
 
 setlocal
 
-set PATH=%PATH%;%CD%\tools
+set PATH=%CD%\tools;%PATH%
 
-set WGET_ARGS=--no-check-certificate
+set WGET_ARGS=-c --no-check-certificate
 set UNZIP_ARGS=-q
```

Comment:
I was surprised to discover that attempting to call `unzip` as e.g.

```
set UNZIP=unzip -q
%UNZIP% ...
```

fails; whereas the current form:

```
set UNZIP_ARGS=-q
unzip %UNZIP_ARGS%
```

works fine. (`wget` seems to handle both invocations correctly.) Such is life in the world of Windows batch scripts, I suppose?